app/Actions/Database/PgBackrestRestore.php

```
@@ -0,0 +1,53 @@
+    public function handle(
+        StandalonePostgresql $database,
+        ?ScheduledDatabaseBackupExecution $execution = null,
+        ?string $targetTime = null
+    ): DatabaseRestore {
+        $attempts = 0;
+        do {
+            $uuid = (string) new Cuid2;
+            $exists = DatabaseRestore::where('uuid', $uuid)->exists();
+            $attempts++;
+            if ($attempts >= 3 && $exists) {
+                throw new \Exception('Unable to generate unique UUID for restore after 3 attempts');
```

PHPMD (2.15.0)

Missing class import via use statement (line '27', column '27'). (undefined)

(MissingImport)